docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+
+```xml
+<exclusive-end id="1161" timestamp="2021-02-26T11:17:25.157" durationms="123.936" />
+```
+
+The `GMP work packet processing` sub-increment continues to run concurrently. The end of this subincrement's operations are recorded using the `<concurrent-end>` element. 
+
+```xml
+<concurrent-end id="1162" type="GMP work packet processing" contextid="1154" timestamp="2021-02-26T11:17:25.469" terminationReason="Work target met">
+<gc-op id="1163" type="mark increment" timems="311.867" contextid="1154" timestamp="2021-02-26T11:17:25.469">
+  <trace-info scanbytes="254708852" />
+</gc-op>
+</concurrent-end>
+```
+
+The child element `<trace-info>` shows that the processing has scanned 242.91 MB (254,708,852 B), which exceeds the 108.25 MB scan target.
+
+The garbage collector now returns to running partial cycles to reclaim free space in the heap before the next global mark phase increment is triggered. To see an example of how a `balanced` partial GC cycle appears in the logs, see the [`Balanced` examples - Partial GC Cycle](vgclog_examples#partial-gc-cycle).
+
+Following some partial GC cycles, an allocation taxation threshold is reached which triggers a STW pause followed by another global mark phase increment. The element `<gc-start>` in the following log excerpt has a `contextid=1154` and type `global mark phase`, which indicates that this is a global mark phase sub-increment associated with our global *mark* cycle example.
+
+```xml
+<exclusive-start id="1175" timestamp="2021-02-26T11:17:28.993" intervalms="1978.886">
+  <response-info timems="5.111" idlems="1.714" threads="2" lastid="00000000006EDE00" lastname="RunDataWriter.1" />
+</exclusive-start>
+<allocation-taxation id="1176" taxation-threshold="1073741824" timestamp="2021-02-26T11:17:28.994" intervalms="1978.879" />
+<gc-start id="1177" type="global mark phase" contextid="1154" timestamp="2021-02-26T11:17:28.994">
+  <mem-info id="1178" free="1451229184" total="4294967296" percent="33">
+    <remembered-set count="3325824" freebytes="158401024" totalbytes="171704320" percent="92" regionsoverflowed="2" regionsstable="0" regionsrebuilding="133"/>
+  </mem-info>
+</gc-start>
+```
+
+The `<allocation-taxation>` element shows the taxation threshold for triggering this global mark phase increment is set to 1024 MB, half of the size of the *eden* space, as expected. 
+
+`<gc-start>` records the heap to have 1384 MB (1,451,229,184 B)of free memory available at the beginning of this global mark phase increment. This compares to the 1376 MB (1,442,840,576 B) of free memory available at the end of the previous global mark phase increment. Although free memory was reclaimed by the partial GC cycles that ran between these global mark phase increments, free memory was allocated to objects when application threads ran, resulting in a net reduction of free memory available.
+
+The `<remembered set>` element shows that there are two overflow regions to rebuild.
```

Comment:
Those 2 regions are overflowed during this second GMP increment. Those ones of those 133 that we were trying to rebuilt, but unfortunately overflowed again, along the  way.

Reply:
Removed the  line:

_The `<remembered set>` element shows that there are two overflow regions to attempt to rebuild._